azure-pipelines.yml

```diff
@@ -148,7 +97,7 @@ jobs:
         trap cleanup EXIT
         echo "$GPG_KEY" | base64 -d > $KEY_FILE
         gpg --homedir $GPG_DIR --no-tty --quiet --import $KEY_FILE
-        cd $(Build.StagingDirectory)/release
+        cd $(Build.StagingDirectory)/release-artifacts/github
```

Comment:
I'm really not sure what kind of directory merging `DownloadPipelineArtifact@0` does if say both `linux-release` and `macos-release` have a `github` folder.

A possible way to test this would be to run a test commit that:
1. Considers itself a release by tricking the check_release job
2. Disabling the Maven/npm push in build-unix
2. Stops the release process after downloading the artifacts, and instead of all the pushing just publishes its `StagingDirectory/release-artifacts` as a new artifact, so we can inspect and check that all the files are there.

Reply:
Sorry should have been clearer, I tested this by adding a dummy step to `build.yml` which downloaded all of them and directories got merged correctly.

Reply:
:+1: